.github/workflows/tos.yaml

```diff
@@ -55,8 +72,8 @@ jobs:
 
   tos-not-accepted:
     runs-on: ubuntu-latest
-    needs: [ parse-issue ]
-    if: needs.parse-issue.outputs.accepted == 'false'
+    needs: [ react-to-new-issue, parse-issue ]
+    if: always() && needs.parse-issue.outputs.accepted == 'false'
```

Comment:
Is this `always() for debugging purpose ? :)

Reply:
We discussed this together. For future ourselves, [always](https://docs.github.com/en/actions/learn-github-actions/expressions#always) allows to always execute the jobs even if its dependencies failed or were canceled.
In our case, react-to-new-issue could have not been executed, so with `always` the job here is always executed.